src/resources/locales/en.json

```diff
@@ -291,26 +291,26 @@
         "remainingTime": "Loans (remaining days)",
         "renewButton": "Renew",
         "returnButton": "Return",
-        "returnedLcp": "This book can't be read because the LCP licence has been returned.",
-        "revokedLcp": "This book can't be read because the LCP licence has been revoked.",
+        "returnedLcp": "This LCP-protected publication is returned.",
```

Comment:
"is" -> "was", no?

Reply:
it works well as a present "state" (the past refers to the "action"). it's a minor semantic nitpick though, not a deal breaker :)